Templating/FilterExtension.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Liip\ImagineBundle\Templating;
 
+@trigger_error('The ' . FilterExtension::class . ' class is deprecated since version 2.2 and will be removed in 3.0; use ' . LazyFilterExtension::class . ' instead.', E_USER_DEPRECATED);
```

Comment:
```suggestion
@trigger_error('The ' . FilterExtension::class . ' class is deprecated since version 2.7 and will be removed in 3.0; use ' . LazyFilterExtension::class . ' instead.', E_USER_DEPRECATED);
```

?